app/google-services.json

```diff
@@ -64,14 +62,7 @@
       ],
       "services": {
         "analytics_service": {
-          "status": 2,
-          "analytics_property": {
-            "tracking_id": "UA-42015512-1"
```

Comment:
I am afraid about these removal of analytics tracking ids. Is this json file generated by firebase?